src/Ledger/PParams.lagda

```diff
@@ -15,13 +23,13 @@ open import Tactic.Derive.DecEq
 
 open EpochStructure es
 \end{code}
-
-Adjustable parameters of the Cardano ledger. They are used in block
-validation and may affect things such as the minimum fees, maximum and
-minimum sizes of certain things and various other features.
-
-\begin{figure*}[h]
+The \AgdaModule{Ledger.PParams} module depends on the \AgdaModule{Ledger.Epoch} module, which defines the epoch
+structure used in the ledger. It also uses some utility functions and imports.
+\begin{figure*}[h!]
+{\small
+\begin{AgdaAlign}
```

Comment:
When multiple blocks of code appear in a single figure, we should enclose all the blocks in a single `AgdaAlign` environment, otherwise, e.g., the field names won't be aligned to the right of the `field` keyword.